### PR TITLE
Terra-Clinical-Header

### PR DESCRIPTION
**Pull request by jmsv6d**
### Summary
A Header component that allows elements to be placed on the left and right ends of the header with a left aligned title in the center. 
 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-clinical-header/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Terra Clinical Header
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-clinical-header.svg)](https://www.npmjs.org/package/terra-clinical-header)
+[![Build Status](https://travis-ci.org/cerner/terra-clinical.svg?branch=master)](https://travis-ci.org/cerner/terra-clinical)
+
+The Terra ClinicalItemView component allows displays to be organized into rows and column and themed, while providing means to add accessory elements.
```
Comment (emilyrohrbough) [inline packages/terra-clinical-header/README.md]: Update description.

File: packages/terra-clinical-header/package.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    "test:nightwatch-chrome": "SPECTRE_TEST_SUITE=terra-clinical-header node ./node_modules/terra-toolkit/lib/scripts/nightwatch.js chrome",
+    "test:nightwatch-firefox": "SPECTRE_TEST_SUITE=terra-clinical-header node ./node_modules/terra-toolkit/lib/scripts/nightwatch.js firefox",
+    "test:nightwatch-safari": "SPECTRE_TEST_SUITE=terra-clinical-header node ./node_modules/terra-toolkit/lib/scripts/nightwatch-non-parallel.js safari",
+    "test:remote": "REMOTE=true node ./node_modules/terra-toolkit/lib/scripts/nightwatch-process.js --config tests/nightwatch.conf.js",
+    "test:remote:all": "REMOTE=true node ./node_modules/terra-toolkit/lib/scripts/nightwatch-process.js --config tests/nightwatch.conf.js --env chrome-tiny,chrome-small,chrome-medium,chrome-large,chrome-huge,chrome-enormous,firefox-tiny,firefox-small,firefox-medium,firefox-large,firefox-huge,firefox-enormous,ie10-tiny,ie10-small,ie10-medium,ie10-large,ie10-huge,ie10-enormous,ie11-tiny,ie11-small,ie11-medium,ie11-large,ie11-huge,ie11-enormous,edge-tiny,edge-small,edge-medium,edge-large,edge-huge,edge-enormous,safari-tiny,safari-small,safari-medium,safari-large,safari-huge,safari-enormous"
```
Comment (emilyrohrbough) [inline packages/terra-clinical-header/package.json]: Add `WEBPACK_CONFIG_PATH=../../../../terra-clinical-site/webpack.config` to the nightwatch tests so they can be run from a package level.

File: packages/terra-clinical-header/docs/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Terra Header
+
+A Header component that allows elements to be placed on the left and right ends of the header with a left aligned title in the center. 
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-header`
```
Comment (dkasper-was-taken) [inline packages/terra-clinical-header/docs/README.md]: Should this be terra-clinical-header?

File: packages/terra-clinical-header/lib/Header.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+.terraClinical-Header {
+  background-color: $terra-grey-5;
+  border-bottom-color: $terra-grey-15;
+  border-bottom-style: solid;
+  border-bottom-width: 1px;
+  padding: 0.714rem;
+  width: 100%;
+
+  .terraClinical-Header--title {
```
Comment (dkasper-was-taken) [inline packages/terra-clinical-header/lib/Header.scss]: Is title intended to be a modifier class? or should it be .terraClinical-Header-title?

**Pull request merged**